src/imap.rs

```diff
@@ -1032,9 +1060,10 @@ impl Imap {
             let (sender, receiver) = std::sync::mpsc::channel();
             let v = self.watch.clone();
 
+            warn!(context, 0, "IMAP-IDLE SPAWNING");
```

Comment:
at one point in my long debugging session i had the impression that IDLE would not be properly interrupted ...

Reply:
maybe not a `warn` though, as this is expected to be happening